### ISSUE TITLE
npm WARN @vueuse/core@2.0.24 requires a peer of @vue/composition-api@^0.3.0 but none is installed

### ISSUE THREAD

**Issue by thearabbit** [antfu/vueuse#75]
I base on 
```js
    "@vue/composition-api": "^0.5.0",
    "@vueuse/core": "^2.0.24",
```

**Comment by antfu** [comment]
Can you try to run `npm i` again and see if the warning still preserves?

**Comment by thearabbit** [comment]
still the same
```js
npm WARN @vueuse/core@2.0.24 requires a peer of @vue/composition-api@^0.3.0 but none is installed. You must install peer dependencies yourself.

audited 5279 packages in 4.982s

30 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by antfu** [comment]
Oh I see, semver treat 0.x version differently. https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

I will make a change address to that. Thanks!

**Comment by thearabbit** [comment]
could close?

**Comment by antfu** [comment]
v0.0.26 released with the fix.

**Comment by thearabbit** [comment]
Wow still the same
```js
npm i @vueuse/core@vue2 @vue/composition-api
npm WARN @vueuse/core@2.0.26 requires a peer of @vue/composition-api@^0.3.0 but none is installed. You must install peer dependencies yourself.

+ @vue/composition-api@0.5.0
+ @vueuse/core@2.0.26
added 2 packages from 2 contributors and audited 5279 packages in 6.706s

30 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by thearabbit** [comment]
My package
```
"dependencies": {
    "@babel/runtime": "^7.9.2",
    "@quasar/extras": "^1.6.4",
    "@u3u/vue-hooks": "^2.0.1",
    "@vue/composition-api": "^0.5.0",
    "@vueuse/core": "^2.0.26",
    "apollo-cache-inmemory": "^1.6.5",
    "apollo-client": "^2.6.8",
    "apollo-link": "^1.2.14",
    "apollo-link-http": "^1.5.17",
    "apollo-server-express": "^2.12.0",
    "bcrypt": "^4.0.1",
    "chart.js": "^2.9.3",
    "decimal.js": "^10.2.0",
    "graphql": "^14.6.0",
    "graphql-load": "^0.1.1",
    "graphql-scalars": "^1.1.0",
    "graphql-subscriptions": "^1.1.0",
    "graphql-tag": "^2.10.3",
    "inputmask": "^5.0.3",
    "lodash": "^4.17.15",
    "merge-graphql-schemas": "^1.7.7",
    "meteor-node-stubs": "^1.0.0",
    "moment": "^2.24.0",
    "mongoose": "^5.9.9",
    "numeral": "^2.0.6",
    "perfy": "^1.1.5",
    "printd": "^1.4.1",
    "quasar": "^1.9.14",
    "shelljs": "^0.8.3",
    "simpl-schema": "^1.5.7",
    "to-css": "^1.2.1",
    "v-click-outside": "^3.0.1",
    "vee-validate": "^3.2.5",
    "vue": "^2.6.11",
    "vue-apollo": "^3.0.3",
    "vue-barcode-scanner": "^0.6.0",
    "vue-chartjs": "^3.5.0",
    "vue-clipboard2": "^0.3.1",
    "vue-count-to": "^1.0.13",
    "vue-json-pretty": "^1.6.3",
    "vue-meta": "^2.3.3",
    "vue-meteor-tracker": "^2.0.0-beta.5",
    "vue-router": "^3.1.6",
    "vue-types": "^1.7.0",
    "vuex": "^3.1.3",
    "vuex-router-sync": "^5.0.0"
  },
```

**Comment by antfu** [comment]
My bad. Fixed in 9ffd6e74386f91b029ebc226060be91021ec760e. Will ship in next release. Thanks

**Comment by thearabbit** [comment]
thanks again 👍
